app/Models/Show.php

```diff
@@ -60,4 +63,22 @@ public function artistTypes()
         return $this->belongsToMany(ArtistType::class);
     }
 
+
+    public function toFeedItem(): FeedItem
+    {
+        return FeedItem::create()
+            ->id($this->id)
+            ->title($this->title)
+            ->description($this->description)
```

Comment:
Ah non! tu ne peux pas changer les méthodes de l'objet FeedItem!
Il n'y a pas de méthodes ->description(), ->poster_url(), etc.

En fait, tu dois fournir aux méthodes prédéfinies les données correspondantes du spectacle. Par exemple, donner la description du spectacle comme summary (résumé).
**->summary($this->description)**


```
return FeedItem::create()
            ->id($this->id)
            ->title($this->title)
            ->summary($this->description)
            ->updated($this->updated_at)    //Gérer le cas ou le champ est null
            ->link($this->link)                        //Fournir l'URL qui affiche le show
            ->authorName('')                         //Peut laisser à '' ou l'auteur du spectacle
            ->authorEmail('');
```